frontend/src/app/store/reducers/app.ts

```diff
@@ -389,6 +414,10 @@ const initialState: AppState = {
   configurations: {
     configurations: undefined,
   },
+  codeTableVersion: (localStorage.getItem("codeTableVersion")!== null) ?
```

Comment:
@Scarlett-Truong is this necessary? Is there a reason the codeTableVersion would live outside the scope of redux and have a separate localState property for tracking the codeTableVersion?

Reply:
Thats why I'm a bit confused by this. You're checking the localState for that value and setting it to the initialState, but if that value isn't there you're defaulting it to ```{ complaintManagement: "1", caseManagement: "1" } ``` and then once you're request for the codeTables is complete you're dispatching the version numbers to redux. The localStorage step seems redundant.